### PR TITLE
Create square_location_id as part of the model definition

### PR DESCRIPTION
**Pull request by Yong-L**
This enables fetching the seller by square_location_id and storing it in
the frontend so that the id can then be referenced when the charge is
made to update square

### CODE REVIEW

**Comment by Yong-L** [comment]
Do you have any reference code testing required fields vs non-required?

**Comment by jtmckibb** [comment]
LOL I can't comment on the comments because Stanley accidentally closed the PR but here: https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/active_model/validate_presence_of_matcher.rb

File: app/models/seller.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Seller < ApplicationRecord
 
   validates_inclusion_of :founded_year, :in => 1800..2020
   validates_uniqueness_of :seller_id
+  validates_uniqueness_of :square_location_id
```
Comment (jtmckibb) [inline app/models/seller.rb:8]: also wanna validate the presence of too :) 

https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/active_model/validate_presence_of_matcher.rb

Comment (Yong-L) [reply app/models/seller.rb:8]: Haha no worries

**Pull request merged**